src/components/learner-credit-management/search/CatalogSearchResults.jsx

```diff
@@ -134,14 +133,10 @@ BaseCatalogSearchResults.propTypes = {
   error: PropTypes.shape({
     message: PropTypes.string,
   }),
-
   searchState: PropTypes.shape({
     page: PropTypes.number,
   }).isRequired,
   paginationComponent: PropTypes.func,
-  // eslint-disable-next-line react/no-unused-prop-types
-  row: PropTypes.string,
-  contentType: PropTypes.string.isRequired,
```

Comment:
[context] As far as I can tell, these props are unused. Was seeing an erroneous "`contentType` prop is required warning."